.circleci/config.yml

```diff
@@ -20,15 +20,20 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            make install
+            make install.build
             # pip install --editable git+https://github.com/openfisca/country-template.git@BRANCH_NAME#egg=OpenFisca-Country-Template  # use a specific branch of OpenFisca-Country-Template
             # pip install --editable git+https://github.com/openfisca/extension-template.git@BRANCH_NAME#egg=OpenFisca-Extension-Template  # use a specific branch of OpenFisca-Extension-Template
 
       - save_cache:
-          key: v1-py3-{{ .Branch }}-{{ checksum "setup.py" }}
+          key: v1-py3-deps-{{ .Branch }}-{{ checksum "setup.py" }}
           paths:
             - /tmp/venv/openfisca_core
 
+      - save_cache:
+          key: v1-py3-build-{{ .Revision }}
```

Comment:
> This is going to be costly in terms of object storage, as we are caching new artifacts for every commit. I am wondering if it would not be less costly in terms of resources to re-run `build` at the deploy step, because that step is triggered on such a small proportion of overall builds that it might outbalance the gain from caching.

Reply:
Indeed.